dev/com.ibm.ws.webcontainer.security.app/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -86,3 +89,9 @@ includePathInWASReqURL.desc=Setting the Path parameter can allow the client/brow
 
 trackLoggedOutSSOCookies=Track logged out LTPA single signon tokens
 trackLoggedOutSSOCookies.desc=Specifies whether to track LTPA single signon tokens that are logged out on a server so that it can not be reused on the same server.
+
+overrideHttpAuthenticationMechanism=Name of the container provided HttpAuthenticationMechanism to override the application configured HttpAuthenticationMechanism. The HttpAutheMechanism is defined by the Java EE 8 Security specification.
```

Comment:
We need to tell the user what "HttpAuthenticationMechanism" is. It sounds redundant to say something like, "HttpAuthenticationMechanism mechanism," but it is beneficial for translation.

Also, I think that "specification" is only capitalized in reference to an API, correct?

Suggested changes:
The name of the container-provided HttpAuthenticationMechanism mechanism that overrides the application-configured HttpAuthenticationMechanism mechanism. The HttpAuthenticationMechanism mechanism is defined by the Java EE 8 Security API specification.

Reply:
HttpAuthenticationMechanism is a module to authenticate incoming http requests.
Since overrideHttpAuthenticationMechanism is a title, it needs to be less than 50 characters, so it is changed as follows: 
Name of HttpAuthenticationMechanism to override